test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -214,6 +223,118 @@ private static void dumpProcessLog(final ProcessBuilder b) {
 		}
 	}
 
+	private static int countStringInFile(String stringToLookFor, File file1) throws FileNotFoundException,IOException {
+		int count = 0;
+		String line = null;
+		String[] buffer;
+
+		BufferedReader bfReader = new BufferedReader(new FileReader(file1));
+		while((line = bfReader.readLine()) != null)   {
+			if (line.contains(stringToLookFor))
+				count++;
+		}
+		logger.info("File: "+file1.getName()+" has "+count+" occurences of the word "+stringToLookFor);
+		bfReader.close();
+		return count;
+        }
+
+	private static boolean checkLogFiles(String fileNamePattern, String stringToLookFor) {
+		boolean foundMatch = false;
+		try{
+			File f = new File(System.getProperty("user.dir"));
+			FilenameFilter filter = new FilenameFilter() {
+				public boolean accept(File f, String name){
+					return name.matches(fileNamePattern);
+				}
+			};
+
+			//List all the log files matching the criteria in the order of last modified
+			File[] files = f.listFiles(filter);
+			Arrays.sort(files, Comparator.comparingLong(File::lastModified).reversed());
+
+			for (int i = 0; i < files.length; i++) {
+				if (countStringInFile(stringToLookFor,files[i]) > 0){
+					foundMatch = true;
+					break;
+				}
+			}
+		} catch (FileNotFoundException e) {
+			System.err.println("FileNotFoundException in checkLogFiles: " + e.getMessage());
+		} catch (IOException e) {
+                        System.err.println("IOException in checkLogFiles: " + e.getMessage());
+		}
+		return foundMatch;
+	}
+
+	private static int getProcessPid(Process p) {
+		int pid = 0;
+		if (p.getClass().getName().equals("java.lang.UNIXProcess")) {
+			try {
+				Field f = p.getClass().getDeclaredField("pid");
+				f.setAccessible(true);
+				pid  = f.getInt(p);
+			} catch (IllegalAccessException | IllegalArgumentException | NoSuchFieldException | SecurityException e){
+				System.err.println("Cant determine the Unix process id in getProcessPid: " + e.getMessage());
+			}
+		}
```

Comment:
If we ever implement JITServer on Windows and zOS, this will not work. It's more robust to have a message or an assert (on the else part) that this version of the test only works on UNIX like OSes.

Reply:
Added the assert.

Reply:
If the only use of the PID is to be able to find the verbose logs then we can probably just not include PIDs in the filenames in the future and use something else that the test harness itself generates, e.g. a simple random number.